src/lib/user_command_input/user_command_input.ml

```diff
@@ -186,14 +188,58 @@ let to_user_command ?(nonce_map = Account_id.Map.empty) ~get_current_nonce
       ~inferred_nonce
     |> Deferred.return
   in
+  let receiver_pk = Signed_command_payload.receiver_pk user_command_payload in
+  let token = Signed_command_payload.token user_command_payload in
+  (* TODO(omerzach): is this the "right" token? *)
+  let receiver : Account_id.t = Account_id.create receiver_pk token in
+  let receiver_account = get_account receiver in
+  (* If the receiver account doesn't exist yet (as far as we can tell) *and* the
+   * user command isn't sufficient to cover the account creation fee, log a
+   * warning. *)
+  let () =
+    match receiver_account with
+    | `Bootstrapping ->
+        () (* TODO(omerzach): What should we do here? *)
+    | `Active (Some _) ->
+        ()
+    | `Active None ->
+        (* TODO(omerzach): What should we do when amount is none? Should we
+         * instead skip this warning? *)
+        let amount : Currency.Amount.t =
+          match Signed_command_payload.amount user_command_payload with
+          | Some amount ->
+              amount
+          | None ->
+              Currency.Amount.zero
+        in
```

Comment:
What should we do when amount is `None`? Should we instead skip the check? I don't understand the nuances of the situations when this is `None`.

Reply:
yeah skip the warning in this case I think, but maybe @mrmr1993 can confirm